Front/category.cpp

```diff
@@ -1,18 +1,57 @@
 #include "category.h"
 #include "ui_category.h"
 #include "addcategory.h"
-#include "popupmenu.h"
+#include "popupmenubutton.h"
 #include "dynamiccategory.h"
 #include "deletednotes.h"
 #include "userpage.h"
 #include "addsavenote.h"
 #include "notesincategory.h"
+#include "ui_notesincategory.h"
+#include "ui_addcategory.h"
+#include "ui_addsavenote.h"
 
 Category::Category(QWidget *parent) :
     QMainWindow(parent),
     ui(new Ui::Category)
 {
     ui->setupUi(this);
+
+    m_addcategoryform.reset(new AddCategory());
+    m_addsavenote.reset(new AddSaveNote());
+    m_notesincategory.reset(new NotesInCategory());
+
+    connect(ui->CCreateCategoryButton, SIGNAL(clicked()), m_addcategoryform.get(), SLOT(show()));
+    connect(ui->CCreateCategoryButton, SIGNAL(clicked()), this, SLOT(close()));
+
+    connect(ui->CreateNoteButton, SIGNAL(clicked()), m_addsavenote.get(), SLOT(show()));
+    connect(ui->CreateNoteButton, SIGNAL(clicked()), this, SLOT(close()));
+
+
+    connect(m_addcategoryform->getAddCategory().ACCreateCategoryButton, SIGNAL(clicked()), m_addcategoryform.get(), SLOT(onButtonSend()));
+    connect(m_addcategoryform->getAddCategory().ACCreateCategoryButton, SIGNAL(clicked()), this, SLOT(show()));
```

Comment:
Don't forget to call clear method on all your window's fields, because after they are being closed and opened again entered data remains

Reply:
Fixed